### PR TITLE
feat: DHIS2-10188 Events GET endpoint in New Tracker

### PR DESCRIPTION
**Pull request by gnespolino**

### CODE REVIEW

File: dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/TrackerControllerSupport.java [hunk 1]
```diff
@@ -30,5 +30,5 @@
 
 public class TrackerControllerSupport
 {
-    public static final String RESOURCE_PATH = "/tracker";
+    public static final String RESOURCE_PATH = "tracker";
```
Comment (enricocolasante) [inline dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/TrackerControllerSupport.java:33]: Are you sure this is correct?
We are using the `/` for any other controller

**Review by enricocolasante — APPROVED** [review]
One comment

File: dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/TrackerControllerSupport.java (hunk 1 above)
Comment (gnespolino) [reply dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/TrackerControllerSupport.java:33]: Spring request mapping will add a / if needed. I'll will double check everything works though

File: dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/domain/web/PagingWrapper.java [hunk 2]
```diff
@@ -29,6 +29,7 @@
  */
 
 import java.util.Collection;
+import java.util.List;
```
Comment (zubaira) [inline dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/domain/web/PagingWrapper.java:32]: Why did we add this?

**Review by zubaira — APPROVED** [review]

File: dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/domain/web/PagingWrapper.java (hunk 2 above)
Comment (gnespolino) [reply dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/domain/web/PagingWrapper.java]: unused. I'll remove it

**Pull request merged**